### DISCUSSION TITLE
Where to find api Key

### DISCUSSION THREAD

**Discussion by algerowalid** [AoElite/claude4j#1] [Q&A] [answered]
Hello, thanks for this project 

But please where to find api Key 

ClaudeAPI client = new ClaudeClient("YOUR_API_KEY")

If even Claude affirme that no api IS available for rhe project 

Thanks you

**Comment by AoElite** [comment, marked as answer]
@algerowalid You can create a API key at https://console.anthropic.com/settings/keys once you have an account.

**Reply by algerowalid** [reply]
You are the best ... thank's very much
